sycl/test-e2e/Graph/device_query.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: cuda || level_zero, gpu
+// REQUIRES: opencl || cuda || level_zero
```

Comment:
Wonder if we could remove this requires all together as well.

Reply:
Yeah this should have been removed with the other tests. Fixed now, cheers.